test/test_http_client.rb

```diff
@@ -68,5 +68,29 @@ def test_http_cookie_with_multiple_cookies_from_one_domain
     def test_http_cookie_with_no_cookies_from_domain
       assert_equal nil, @client.http_cookie('RETS-Session-ID')
     end
+
+    def test_save_cookie_store
+      cookie_file = Tempfile.new('cookie_file').path
+
+      #setup cookie store
+      http_a = HTTPClient.new
+      http_a.set_cookie_store(cookie_file)
+      client_a = Rets::HttpClient.new(http_a, { cookie_store: cookie_file }, nil, "http://rets.rets.com/somestate/login.aspx")
+
+      #add cookie
+      cookie = "RETS-Session-ID=879392834723043209; path=/; domain=rets.rets.com; expires=Wednesday, 31-Dec-2037 12:00:00 GMT"
+      http_a.cookie_manager.parse(cookie, URI.parse("http://www.rets.rets.com"))
+
+      #save cookie
+      client_a.save_cookie_store
```

Comment:
If any-one wants to see this failing on master than you need `client_a.save_cookie_store(true)` here. However I have removed this option with this PR (because there is no concept of `force` for the `HTTPClient::CookieManager`